src/modules/imageresizer/README.md

```diff
@@ -0,0 +1,43 @@
+<center>
+
+<img width="200" src="../../../doc/images/Logo.jpg">
+
+# Image Resizer
+> A Windows Shell Extension for bulk image resizing
+
+[**Overview**](#overview) · 
+[**Settings**](#settings)
+</center>
+
+## Overview
+Image Resizer is a windows shell extension for bulk image resizing. After installing PowerToys, right-click on one or more selected image files in File Explorer, and then select _Resize pictures_ from the menu.
+
+<figure>
+  <img src="../../../doc/images/imageresizer/resizeNormal.gif" alt="Image Resizer Demo" >
+  <figcaption>Figure 1 : Image Resizer Demo</figcaption>
+</figure>
+
+Image Resizer also allows you to resize images by dragging and dropping your selected files with the right mouse button. This allows you to save your resized pictures in another folder.
+
+<figure>
+  <img src="../../../doc/images/imageresizer/resizeDragAndDrop.gif" alt="Image Resizer Drag And Drop Demo" >
+  <figcaption>Figure 2 : Image Resizer Drag And Drop Demo</figcaption>
+</figure>

```

Comment:
@arjunbalgovind Can you replace the html with raw markdown?  This won't preview for me in the pull request tool on github.  I'm not sure if that's why, but generally it's preferable to avoid html in markdown if possible.

Reply:
I'll make that change.